types/matter-js/index.d.ts

```diff
@@ -1175,6 +1176,135 @@ declare namespace Matter {
         static shift(bounds: Bounds, position: Vector): void;
     }
 
+    /**
+     * The `Matter.Collision`module contains methods for detecting collisions between a given pair of bodies.
+     *
+     * For efficient detection between a list of bodies, see `Matter.Detector` and `Matter.Query`.
+     *
+     * See `Matter.Engine` for collision events.
+     *
+     * @class Collision
+     */
+    export class Collision {
+        /**
+         * The first body part represented by the collision (see also `collision.parentA`).
+         *
+         * @property bodyA
+         * @type body
+         */
+        bodyA: Body
+
+        /**
+         * The second body part represented by the collision (see also `collision.parentB`).
+         *
```

Comment:
Is there a code-style guideline for blank lines in the docs?

At least it is very in-consistent in this file. And well, due to broken-window-theory, this will just get worse if nobody cleans this up in one go ;-)

I could do that in case one could point me to what the preference is. Or does it simply not matter at all?

Reply:
Take a look at .pretterrc.json in the root. I didn't set it up, and I don't use prettier enough to know how to apply it, but that is the standard we hope people use.

However, it should wait until a separate PR to fix formatting.